directory/cli/src/host.py

```diff
@@ -0,0 +1,170 @@
+import click
+from collections import OrderedDict
+
+import list_command
+from list_command import \
+	field_with_same_name, \
+	field_with_name, \
+	hostgroup_with_host_hgid
+import ipa_wrapper_command
+import ipa_utils
+import appliance_cli.text as text
+import appliance_cli.utils
+import utils
+from exceptions import IpaRunError
+from option_transformer import OptionTransformer
+
+HOST_LIST_FIELD_CONFIGS = OrderedDict([
+	('Host name', field_with_same_name),
+	('Password', field_with_same_name),
```

Comment:
Host passwords are a little different to user passwords so worth investigating _if_ these can be seen with `host-show`, if not then it's no problem to remove password from the display fields